src/rpc/misc.cpp

```diff
@@ -39,6 +40,8 @@ static UniValue validateaddress(const JSONRPCRequest& request)
             "  \"iswitness\" : true|false,     (boolean) If the address is a witness address\n"
             "  \"witness_version\" : version   (numeric, optional) The version number of the witness program\n"
             "  \"witness_program\" : \"hex\"     (string, optional) The hex value of the witness program\n"
+            "  \"error\" : \"message\"           (string, optional) The error message if the address is an invalid bech32 address\n"
```

Comment:
f877a9b4af7da6632430eff898b957107514e6de

Could include only when prefix matches `CChainParams::Bech32HRP`?

Reply:
I thought about this but I wasn't sure, because one of the checks is that the HRP is present and valid too. You're probably right, that would be better for the user.